jira_access/_jira_client.py

```diff
@@ -46,15 +46,19 @@ def _execute_http_post_requests_from_json_file(self,
                                                    json_file_path: str,
                                                    api_action: str,
                                                    params: Optional[Mapping[str, str]] = None,
-                                                   headers: Optional[Mapping[str, str]] = None) -> Json | JsonType:
-        with open(json_file_path, "rb") as data:
-            return self._execute_http_post_request(params=params, data=data, api_action=api_action, headers=headers)
+                                                   headers: Optional[Mapping[str, str]] = None) -> JsonType:
+        with open(json_file_path, "r", encoding="utf-8") as f:
+            data = json.load(f)
+        return self._execute_http_post_request(params=params,
+                                               data=json.load(data),
```

Comment:
isn't this just
```suggestion
                                               data=data,
```
?
data is already a json object (data = json.load(f))

Reply:
Yes you are completely right!